stdlib/src/builtin/string.mojo

```diff
@@ -1450,44 +1560,43 @@ struct String(
 
         Returns:
             True if the String is one of the whitespace characters
-                listed above, otherwise False."""
+                listed above, otherwise False.
+        """
         # TODO add line and paragraph separator as stringliteral
         # once unicode escape secuences are accepted
-        # 0 is to build a String with null terminator
-        alias information_sep_four = List[UInt8](0x5C, 0x78, 0x31, 0x63, 0)
-        """TODO: \\x1c"""
-        alias information_sep_two = List[UInt8](0x5C, 0x78, 0x31, 0x65, 0)
-        """TODO: \\x1e"""
-        alias next_line = List[UInt8](0x78, 0x38, 0x35, 0)
+        var next_line = List[UInt8](0xC2, 0x85)
         """TODO: \\x85"""
-        alias unicode_line_sep = List[UInt8](
-            0x20, 0x5C, 0x75, 0x32, 0x30, 0x32, 0x38, 0
-        )
+        var unicode_line_sep = List[UInt8](0xE2, 0x80, 0xA8)
         """TODO: \\u2028"""
-        alias unicode_paragraph_sep = List[UInt8](
-            0x20, 0x5C, 0x75, 0x32, 0x30, 0x32, 0x39, 0
-        )
+        var unicode_paragraph_sep = List[UInt8](0xE2, 0x80, 0xA9)
         """TODO: \\u2029"""
 
         @always_inline
-        fn compare(item1: List[UInt8], item2: List[UInt8], amnt: Int) -> Bool:
-            var ptr1 = DTypePointer(item1.unsafe_ptr())
-            var ptr2 = DTypePointer(item2.unsafe_ptr())
+        fn compare(
```

Comment:
Q: should this be part of the public API? If not, could you please rename it to `_compare`. Otherwise, could you please add a docstring?

Reply:
I thought it was irrelevant since it's a helper func declared within the "private" scope of the main body. But ok I'll just but an _ in front

Reply:
Oh I see, I thought for some reason that this was a function on `String` 🤦. It's fine either way, thanks!